Kentico.Kontent.Delivery/DeliveryClient.cs

```diff
@@ -368,7 +383,12 @@ private static void ValidateItemsFeedParameters(IEnumerable<IQueryParameter> par
 
         private async Task<IList<object>> GetLinkedItemsAsync(JObject content)
         {
-            var items = ((JObject)content["modular_content"]).Values().Select(async source => await ModelProvider.GetContentItemModelAsync<object>(source, content["modular_content"]));
+            var items = ((JObject)content["modular_content"])?.Values()?.Select(async source => await ModelProvider.GetContentItemModelAsync<object>(source, content?["modular_content"]));
+
+            if(items == null)
```

Comment:
why not inline these in the return statement? :)

Reply:
Fixed